src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -361,7 +361,11 @@ export class SearchView extends ViewPane {
 		this._register(this.onDidChangeBodyVisibility(visible => this.onVisibilityChanged(visible)));
 	}
 
-	private onVisibilityChanged(visible: boolean): void {
+	protected callRenderBody(parent: HTMLElement): void { // {{SQL CARBON EDIT}}
```

Comment:
@Charles-Gagnon I wasn't able to call renderBody method on the super-super class even though it's protected for some reason. So I added this method to bypass calling the super.renderBody but call super.super.renderBody. Not sure if this is the right way.

Reply:
I'd just call `ViewPane.prototype.renderBody.call(this, parent)` from the NotebookSearchView. What you have is fine generally (and is preferred if you need to do this thing in the future) but let's avoid touching vs stuff as much as we can.

Reply:
I tried this ViewPane.prototype.renderBody.call(this, parent) but ViewPane.prototype is not able to resolve renderBody, that's why I had to the above.

Reply:
Ah, the protected part is making it so that doesn't work. Yeah your solution seems fine then.